_data/simple-icons.json

```
@@ -6454,6 +6454,11 @@
             "hex": "3F5DFF",
             "source": "https://loopback.io/resources"
         },
+        {
+            "title": "Lootcrate",
```

```suggestion
            "title": "Loot Crate",
```

The brand name is stylized as 'Loot Crate' on their website.